indico/modules/rb_new/schemas.py

```diff
@@ -139,6 +141,25 @@ def has_managed_rooms(self, user):
         return has_managed_rooms(user)
 
 
+class CreateBookingSchema(Schema):
+    start_dt = fields.DateTime(required=True)
+    end_dt = fields.DateTime(required=True)
+    repeat_frequency = EnumField(RepeatFrequency, required=True)
+    repeat_interval = fields.Int(missing=0, validate=lambda x: x >= 0)
+    room_id = fields.Int(required=True)
+    user_id = fields.Int()
+    booking_reason = fields.String(load_from='reason', validate=validate.Length(min=3))
+    is_prebooking = fields.Bool(missing=False)
+
+    class Meta:
+        strict = True
+
+    @validates_schema
+    def validate_dts(self, data):
+        if data['start_dt'] >= data['end_dt']:
+            raise ValidationError('Booking end date and time must be greater than start date and time')
```

Comment:
`_(...)` if users can see this (but if client-side validation usually covers it nevermind)

also, I'd go for something concise, like "Booking cannot end before it starts"